test/models/activities_fetcher_test.rb

```diff
@@ -12,7 +12,8 @@ def setup
       headers: {content_type: "application/json; charset=utf-8"}
     )
 
-    stub_get('/repos/prasadsurase/code-curiosity').to_return(
+    stub_get('/repositories/67219068').to_return(
+      # all activities are related to code-curasity repo. that's why it is manually stubbed here.
```

Comment:
1. Typo "code-curasity"
2. It is not clear what you mean. Please rewrite the comment